test/setup/deployers/operator-openshift.ts

```diff
@@ -10,10 +10,8 @@ async function deployKubernetesMonitor(
 ): Promise<void> {
   await kubectl.applyK8sYaml('./test/fixtures/operator/operator-source.yaml');
   await kubectl.applyK8sYaml('./test/fixtures/operator/installation.yaml');
-
   // Await for the Operator to become available, only then
   // the Operator can start processing the custom resource.
-  await kubectl.waitForDeployment('snyk-operator', 'snyk-monitor');
-
+  await kubectl.waitForDeployment('snyk-operator', 'openshift-marketplace');
```

Comment:
I think this isn't quite right, we want the Operator to appear in the `snyk-monitor` namespace -- this is what would happen if the user installed it through the OpenShift UI

Reply:
Definitely! I'm making sure that all the integration tests keep running and passing.